addons/docs/src/blocks/Source.tsx

```diff
@@ -94,7 +94,12 @@ export const getSourceProps = (
   sourceContext: SourceContextProps
 ): PureSourceProps & SourceStateProps => {
   const { id: currentId, storyById } = docsContext;
-  const { parameters } = storyById(currentId);
+  let parameters = {} as Parameters;
+  try {
+    ({ parameters } = storyById(currentId));
+  } catch (err) {
+    // TODO: in external mode, there is no "current"
```

Comment:
We probably want to get rid of currentId in Docs2 entirely as a follow-up. WDYT?

Reply:
Yeah, I wasn't sure what this is for but I figured we'd do another pass over these components later in the piece.